doc/paperclip.md

```diff
@@ -695,6 +695,7 @@ s3.upload(io, "object/destination/path")
 The Shrine storage has no replacement for the `:url` Paperclip option, and it
 isn't needed.
 
+[metadata_attributes]: https://github.com/shrinerb/shrine/blob/master/lib/shrine/plugins/metadata_attributes.rb
```

Comment:
Thanks for the PR. It would be better to link this to `metadata_attributes` plugin documentation instead of the source code... link to this: https://shrinerb.com/docs/plugins/metadata_attributes